core/templates/dev/head/pages/exploration_editor/editor_tab/teach_oppia_modal_directive.html

```diff
@@ -1,55 +1,66 @@
-<div class="modal-header">
-  <h3>Enter a Sample Answer</h3>
-</div>
+<style>
+.unresolved-answers-loading-text {
```

Comment:
Can we prefix all of these with the directive name, to make sure that these CSS classes are always scoped only to this modal? (maybe teach-oppia-modal if that's the directive name, or add a .teach-oppia-modal class to line 34)

Reply:
Actually, this is not a directive (its modal that JS generates upon clicking button). I will add `teach-oppia-modal` class on line 34.

Reply:
Done.